extensions/sql-bindings/README.md

```diff
@@ -8,6 +8,24 @@ Microsoft SQL Bindings for VS Code enables users to develop Azure Functions with
 
 This extension is bundled into the `SQL Server (MSSQL)` extension for VS Code and will be installed automatically when that extension is updated or installed.
 
+## Getting Started with SQL Bindings
+> [!NOTE]
+> Currently, only C# Azure Functions are supported.
```

Comment:
@Charles-Gagnon not sure if this is rendering right here but it is intended to work similar to this: https://docs.microsoft.com/en-us/contribute/markdown-reference#alerts-note-tip-important-caution-warning

Reply:
I don't think Github supports anything like that, at least I've never seen it.